### PR TITLE
Correctly mark groovy and java source folders

### PR DESCRIPTION
**Pull request by jenniferboedker**
Many thanks for contributing to QUBE!

**Description of changes**
The current version of qube is creating portlets where the source folder is not marked. Furthermore, the portlet cannot run error-free because the notation from java (@component annotation) cannot be translated to groovy. This PR updates the POM to recognize both, groovy and java, as source folders and fixes the portlet template such that it produces a template that works from scratch.

**Technical details**
The source folders are specified in the POM. Furthermore, some specifications have been added to the `build-helper-maven-plugin` plugin to enable maven in finding the groovy source folder.

### CODE REVIEW

**Comment by jenniferboedker** [comment]
[Transfer example portlet into qube template generation](https://trello.com/c/nK6fbOkj/14-transfer-example-portlet-into-qube-template-generation)

File: qube/create/templates/portlet/portlet_groovy_osgi/{{ cookiecutter.project_slug }}/pom.xml [hunk 1]
```diff
@@ -56,7 +56,7 @@
             </snapshots>
             <id>nexus-releases</id>
             <name>QBiC Releases</name>
-            <url>https://qbic-repo.am10.uni-tuebingen.de/repository/maven-releases</url>
+            <url>https://qbic-repo.qbi.uni-tuebingen.de/repository/maven-releases</url>
```
Comment (sven1103) [inline qube/create/templates/portlet/portlet_groovy_osgi/{{ cookiecutter.project_slug }}/pom.xml:59]: qbic

File: qube/create/templates/portlet/portlet_groovy_osgi/{{ cookiecutter.project_slug }}/pom.xml [hunk 2]
```diff
@@ -175,11 +175,41 @@
 	</dependencies>
 
     <build>
+        <scriptSourceDirectory>src/main/groovy</scriptSourceDirectory>
+        <sourceDirectory>src/main/java</sourceDirectory>
+        <testSourceDirectory>src/test/groovy</testSourceDirectory>
+
```
Comment (sven1103) [inline qube/create/templates/portlet/portlet_groovy_osgi/{{ cookiecutter.project_slug }}/pom.xml:181]: great! 🎉

File: qube/create/templates/portlet/portlet_groovy_osgi/{{ cookiecutter.project_slug }}/pom.xml (hunk 1 above)
Comment (jenniferboedker) [reply qube/create/templates/portlet/portlet_groovy_osgi/{{ cookiecutter.project_slug }}/pom.xml:59]: 🙈

Comment (jenniferboedker) [reply qube/create/templates/portlet/portlet_groovy_osgi/{{ cookiecutter.project_slug }}/pom.xml:59]: can you make a suggestion?

**Review by sven1103 — CHANGES_REQUESTED** [review]
Only some minor request, otherwise really good!

File: qube/create/templates/portlet/portlet_groovy_osgi/{{ cookiecutter.project_slug }}/pom.xml (hunk 1 above)
Comment (sven1103) [reply qube/create/templates/portlet/portlet_groovy_osgi/{{ cookiecutter.project_slug }}/pom.xml:59]:
```suggestion
            <url>https://qbic-repo.qbic.uni-tuebingen.de/repository/maven-releases</url>
```

**Review by sven1103 — APPROVED** [review]
LGTM!

**Pull request merged**